finder-app/Makefile

```diff
@@ -0,0 +1,13 @@
+
+#Makefile for ARM64 cross compilation
+
+CROSS_COMPILE = 
+cc = gcc
```

Comment:
@suja8742 convention is to use capital `CC` for this variable, see https://www.gnu.org/software/make/manual/html_node/Implicit-Variables.html